script/1_0_2current_en_script/Arcueid/Day 4/04_00_ARC04_9_1.txt

```diff
@@ -22,26 +22,26 @@ An all-enveloping space of complete silence that made me want to believe that th
 <Page5>
 "......There's nothing unusual, is there?"
 I walk down the corridor.
-The sweat on my forehead coalesce into droplets and fall.
-It is so overly quiet that I fall under the illusion that I'm a foreign entity.
+The sweat on my forehead coalesces into droplets that roll down my face.
+It is so overly quiet that I start to believe that I'm a foreign entity.
 Foreign... foreign... foreign...
 It is customary for vermin that disturb the atmosphere of a place to be crushed.
 <Page6>
 "―――Hah―――"
 Once again, I take a deep breath to calm down.
 I adjust my grip on the knife, wrapping my hand around the handle to prevent it from slipping due to my sweat.
 ...I feel nauseous.
-There's something on the side of my head.
-Nerves seem to extend from the back of my eyeballs to my brain.
+There's a sensation at my temple.
+As though the nerves between the back of my eyes and my brain are being stretched.
 <Page7>
 Are my eyeballs trying to sink into my brain?
 Or is my brain trying to swallow my eyeballs?
-Biting down on the dull pain, I continue walking down the fiery path.
+Biting down on the dull pain, I continue walking along the scorching path.//鈍重な痛みをかみ殺しながら、猛暑の<路|みち>を歩いて行く。
```

Comment:
Fiery is totally valid here, but I worried it might be read as literal, since there _is_ a lot of fire going around.

Reply:
scorching to my eyes should be "scorched" which is a little strange. I kinda like 'fiery' more here.

Reply:
Hmm, scorching heat reads well to me but maybe it's en-GB vernacular